tests/test_modeling_tf_common.py

```diff
@@ -1753,12 +1748,20 @@ def _generate_and_check_results(model, inputs_dict):
             config.eos_token_id = None  # Generate until max length
             config.do_sample = False
 
+            # extract the input to the model
+            is_input_ids = "input_ids" in inputs_dict
+            is_input_features = "input_features" in inputs_dict
+            if not (is_input_ids or is_input_features):
+                raise ValueError("No valid generate input found in inputs_dict")
+            inputs = inputs_dict["input_ids"] if is_input_ids else inputs_dict["input_features"]
+
             # fix config for models with additional sequence-length limiting settings
+            seq_len = inputs.get_shape()[1]
```

Comment:
Not sure if input_features also has its sequence length at 1. Just submitting this quickly before I don't have time.